File: 1-Beginner/1008/bee1008.java
// 1008 - Salário
import java.io.IOException;
import java.util.Scanner;

class bee1008 {

    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);

        int num, qtHoras;
        double valorHora;

        num = scan.nextInt();
        qtHoras = scan.nextInt();

        valorHora = scan.nextDouble();

        System.out.println("NUMBER = " + num);
        System.out.printf("SALARY = U$ %.2f\n", valorHora*qtHoras);

        scan.close();
    }
}

File: 1-Beginner/1008/bee1008.cpp
// 1008 - Salário
#include <iostream>
#include <iomanip>

int main(void) {

    int num, qtHoras;
    double valorHora;

    std::cin>>num>>qtHoras>>valorHora;

    std::cout<<std::fixed<<std::setprecision(2);

    std::cout<<"NUMBER = "<<num<<std::endl<<"SALARY = U$ "<<qtHoras*valorHora<<std::endl;

}